### PR TITLE
Add osu!taiko SR change newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
Note: most of this is paraphrased from [morth1](https://docs.google.com/document/d/1JjHSh6Gzigja0thJDiHwW4E1oH3fxcS0Pze5HJXLErk/edit#)'s proposal document with only cursory understanding, if at all.

Attempted to simplify it as much as possible for newspost material, but as soon as the math script gets busted out, a lot of that goes out the window. Please correct if things aren't right!

### CODE REVIEW

File: news/2020-09-15-changes-to-osutaiko-star-rating.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+#### Speed
+
+In addition, each object gains a bonus weight added on to its object strain depending on the duration of the two most recent note-pairs. To demonstrate briefly with numbers, a note-pair coupling with a duration of 25ms is awarded roughly 7 times more bonus score than a coupling with 130ms between them. This reflects the impact of speed.
+
+#### Penalizing Mashing
+
+Certain mapping patterns that are easily circumvented by players using techniques commonly referred to as 'mashing' have been nerfed specifically to reflect the true difficulty encountered. The following patterns in particular are affected:
+
+- Sequences of more than 12 notes, comprised of repetitions along a 1-4 note long pattern
+- Sequences of more than 16 notes, where every second note is a ka
```
Comment (Raidencio) [inline news/2020-09-15-changes-to-osutaiko-star-rating.md:50]: ka → kat

File: news/2020-09-15-changes-to-osutaiko-star-rating.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+This is most prominent in converted maps, which tend to have very little colour variance, and a select few maps designed to exploit this fact.
+
+A scaling penalty to stamina difficulty is applied to maps with low colour variance, bringing them more in line with their actual skill level.
+
+### Calculating the Result
+
+A fair bit of mathematical nuance goes into actually calculating the end result. If you're interested in the particulars, consult [morth1's proposal document](https://docs.google.com/document/d/1JjHSh6Gzigja0thJDiHwW4E1oH3fxcS0Pze5HJXLErk/edit#) for the full details.
+
+Importantly, Star Rating has been scaled to ensure that most kantan are below 2*, most futsuu are below 2.8*, and most muzukashii are below 4*. In addition, high SR values have been scaled specifically to ensure that beatmaps with FCs or near-FCs are all below 9.5*.
```
Comment (Raidencio) [inline news/2020-09-15-changes-to-osutaiko-star-rating.md:95]: capitalize the beginning of each difficulty name (Kantan, Futsuu, Muzukashii)

**Comment by bdach** [comment]
I could give this a review pass seeing that I've done work on the code, but it might be a few hours before I'm able to.

**Comment by peppy** [comment]
No hurry. I still need to get things ready to begin the deployment, probably won't be ready until tomrorow.

File: news/2020-09-15-changes-to-osutaiko-star-rating.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+Across the board, all objects have been given a base object strain value of 1. This makes dense patterns more rewarding.
+
+#### Speed
+
+In addition, each object gains a bonus weight added on to its object strain depending on the duration of the two most recent note-pairs. To demonstrate briefly with numbers, a note-pair coupling with a duration of 25ms is awarded roughly 7 times more bonus score than a coupling with 130ms between them. This reflects the impact of speed.
+
+#### Penalizing Mashing
+
+Certain mapping patterns that are easily circumvented by players using techniques commonly referred to as 'mashing' have been nerfed specifically to reflect the true difficulty encountered. The following patterns in particular are affected:
```
Comment (lolxdhi1) [inline news/2020-09-15-changes-to-osutaiko-star-rating.md:47]: maybe replace mashing with rolling as it's the proper term

**Comment by bdach** [comment]
I've read this through and there is some missing relevant info that I think is human-understandable (which also isn't in the doc, but is in the actual code), and I felt that the current iteration read a little awkward in places. To avoid spamming github with review comments I figured [I'll just push a commit](https://github.com/bdach/osu-wiki/blob/dec32ecf3a15c07c79cb3e968d0f5adfdc4cdd43/news/2020-09-15-changes-to-osutaiko-star-rating.md) with some proposed changes to my fork. [Here's a readable version](https://github.com/bdach/osu-wiki/blob/dec32ecf3a15c07c79cb3e968d0f5adfdc4cdd43/news/2020-09-15-changes-to-osutaiko-star-rating.md); feel free to pick and choose from the changes.

Sorry if overstepping. Not a native so wording might be bad, and I'll admit freely I haven't read the ASC very thoroughly (I'm guessing for example that my use of italics is not correct). Source of most added info is https://github.com/ppy/osu/pull/9294, which as I mentioned I did clean-up and documentation detail on.

**Comment by Ephemeralis** [comment]
@bdach pulled your version in, looks good!

**Pull request merged**